tasks/internal_reconcile_report_mismatch/schemas/output.json

```diff
@@ -0,0 +1,95 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/internal_reconcile_report_mismatch/schemas/output.json",
+    "title": "internal_reconcile_report_mismatch Lambda Output",
+    "description": "The output for the internal_reconcile_report_mismatch Lambda.",
+    "type": "object",
+    "required": [
+      "jobId",
+      "anotherPage",
+      "mismatches"
+    ],
+    "properties": {
+        "jobId": {
+            "type": "integer",
+            "description": "The unique ID of the reconciliation job."
+          },
+        "anotherPage": {
+          "description": "Indicates if more results can be retrieved on another page.",
+          "type": "boolean"
+        },
+        "mismatches": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "required": [
+              "collectionId",
+              "granuleId",
+              "fileName",
+              "keyPath",
+              "cumulusArchiveLocation",
+              "orcaEtag",
+              "s3Etag",
+              "orcaLastUpdate",
+              "s3LastUpdate",
+              "orcaSizeinBytes",
+              "s3SizeInBytes",
+              "discrepancyType"
+
+            ],
+          "properties": {
+            "collectionId": {
+              "type": "string",
+              "description": "Cumulus Collection ID value from the ORCA catalog."
+            },
+            "granuleId": {
+                "type": "string",
+                "description": "Cumulus granuleID value from the ORCA catalog."
+              }, 
+            "filename": {
+                "type": "string",
+                "description": "Filename of the object from the ORCA catalog."
+              },
+            "keyPath": {
+                "type": "string",
+                "description": "key path and filename of the object in the ORCA catalog."
+              },
+            "cumulusArchiveLocation": {
+              "type": "string",
+              "description": "Expected S3 bucket the object is located in Cumulus. From the ORCA catalog."
+            },       
+            "orcaEtag": {
+                "type": "string",
+                "description": "etag of the object as reported in the ORCA catalog."
+              },
+            "s3Etag": {
+              "type": "string",
+              "description": "etag of the object as reported in the S3 bucket"
+            },
+            "orcaLastUpdate": {
```

Comment:
Wondering if this shouldn't be a EPOCH date/time format since it will likely need to interface with Cumulus web pages.

Reply:
We specify this in other schemas as `The time, in milliseconds since 1 January 1970 UTC, that yada yada`

Reply:
followed the similar format as seen in schema  https://github.com/nasa/cumulus-orca/blob/259c41c9ed6bfb8450b25e746bafd792c2ef73cf/tasks/internal_reconcile_report_phantom/schemas/output.json#L55

Reply:
updated

Reply:
Gah. Just realized that you're returning a string instead of number. So we didn't miss it first time, you were using the standard format there and here.

Reply:
reverted back to previous changes